Makefile

```diff
@@ -70,38 +70,69 @@ start_build:
 $(SITEBUILDDIR)/%.svg: $(SOURCEDIR)/%.svg
 	$(SCOUR) $(SCOUR_FLAGS) $< $@
 
-build: clean start_build build_images build_spec sphinx-build-docs
+build: clean start_build build_images build_spec build_docs
 
 build_images: $(SITEBUILDDIR)/$(OBJECTS)
 
 build_spec: build_spec_rfc build_spec_w3c
 
+# PS and RTF support are provided using a combination of wkhtmltopdf, pdf2ps and/or GNU enscript.
+requirements_xml2rfc:
+	sudo apt install xml2rfc enscript
+	pip install 'pycairo>=1.18' 'weasyprint<=0.42.3'
+
 # https://github.com/mmarkdown/mmark
 # sudo apt install -y mmark
 requirements_mmark:
 	wget https://github.com/mmarkdown/mmark/releases/download/v2.2.25/mmark_2.2.25_linux_amd64.tgz
```

Comment:
Right now it's workable only on linux. My tech debt. Also thinking to make it cross platform

Reply:
ok, done. not sure if it works ... we'll see